test/stickerbook.test.js

```diff
@@ -290,6 +290,32 @@ describe('Stickerbook', () => {
     stickerbook.off('mouse:down', callback);
   });
 
+  it('cannot place a sticker without a position', (done) => {
+    const stickerbook = createStickerbook();
+
+    stickerbook.setSticker('http://www.example.com/images/A.png')
+      .then(() => {
+        try {
+          stickerbook.placeImage({});
+        } catch(e) {
+          return done(); // we want it to err!
+        }
+
+        done('Should have not allowed an empty position');
+      })
+  })
+  
+  it('can place a sticker with a position', (done) => {
+    const stickerbook = createStickerbook();
+
+    stickerbook.setSticker('http://www.example.com/images/A.png')
+      .then(() => {
+        stickerbook.placeSticker({ x: 0, y: 0 });
+        done();
+      })
+      .catch(done);
```

Comment:
The idea is that if you pass something to done, ala `done('oh no')`, mocha will assume that the test failed. So, I'm just forwarding the error message from the `catch` back to mocha. In short, we do need this so that mocha detects the test as failing and not timing out